website/docs/guides/data_source_filters.html.markdown

```diff
@@ -0,0 +1,283 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: data source filters"
+sidebar_current: "docs-vcd-guides-filters"
+description: |-
+  Provides guidance on filters.
+---
+
+
+## Retrieving a data source by filter
+
+Supported in provider *v2.9+*
+
+Several data sources can be retrieved using a filter.
+The criteria should be specific enough to retrieve a single item. The retrieval will fail if the criteria match more than
+one item.
+
+When you don't know the name, you may get the data source using the `filter` section, and using one or more of the following criteria:
+
+* `name_regex` (Optional) matches the name using a regular expression.
+* `date` (Optional) is an expression containing an operator (`>`, `<`, `>=`, `<=`, `==`) and a date. Several formats 
+  are recognized, but one of `yyyy-mm-dd [hh:mm[:ss]]` or `dd-MMM-yyyy [hh:mm[:ss]]` is recommended.
```

Comment:
When you read this for the first time there's some fuzziness because of the word "containing". It looks like we should change this to "begins with" (an operator). And if that's true, we should do this change across all the docs.

Also, it's not clear (or I have missed it) is how to filter by date **range**, where date lands between two dates. If it's possible, we should write it up, if it's not possible, we can live with it for now.

Reply:
Updated docs

Reply:
Thanks. We should use the enhanced description in the data source pages too, now there's discrepancy and if you read data source page you may not read the guide.

Should we also explicitly mention that the operator `==` requires precision down to the microsecond level?